src/util/debounced.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Ensures a function is only run only once every delay
+ * @param delay {number} the number of milliseconds to delay the function call
+ * @return function
```

Comment:
this should be `@returns {Function}` i believe https://jsdoc.app/tags-returns.html